maven-java/akkaserverless-maven-archetype-event-sourced-entity/src/main/resources/archetype-resources/docker-compose.yml

```diff
@@ -10,7 +10,11 @@ services:
     environment:
 #[[      USER_FUNCTION_HOST: ${USER_FUNCTION_HOST:-host.docker.internal}
       USER_FUNCTION_PORT: ${USER_FUNCTION_PORT:-8080}
-      PUBSUB_EMULATOR_HOST: gcloud-pubsub-emulator ]]#
+      PUBSUB_EMULATOR_HOST: gcloud-pubsub-emulator
+      # Uncomment to disable the JWT dev secret
+      # JWT_DEV_SECRET: "false"
+      # Uncomment to set the JWT dev secret issuer
+      # JWT_DEV_SECRET_ISSUER: "my-issuer" ]]#
```

Comment:
I'm now intrigued by this `#[[` and `]]#`. What is this good for? And why we are closing it in the last comment? 

Shouldn't we close it just after `gcloud-pubsub-emulator`?

Reply:
Isn't it to leave variables defined with ${} as is instead of replacing with values when maven archetype creates a project out of the template?

Reply:
Yeah I'm pretty sure it's got something to do with maven archetype templating. I've noticed the markdown files have it around anything that contains `#`, so it's probably needed around this stuff.

Reply:
I tested it with the jwt comments outside it and it works.

Reply:
> Yeah I'm pretty sure it's got something to do with maven archetype templating.

Yes, they're Velocity templates (https://velocity.apache.org/engine/1.7/user-guide.html#literals)

> I've noticed the markdown files have it around anything that contains `#`, so it's probably needed around this stuff.

They're needed around `##` but not around `#` (https://velocity.apache.org/engine/1.7/user-guide.html#comments)

Reply:
It might make sense to put it around the whole file, since this file doesn't need to be templated.